README.md

```diff
@@ -22,6 +22,9 @@ that lives behind an OVS bridge and has its own unique interfaces.
 #### Step 1: Clone the project:
 
 ```
+$ export GOPATH=`pwd`
```

Comment:
I'm thinking being explicit is better for beginners who are not as familiar with golang..
Folks who are familiar with golang will know how to ignore the GOPATH part..

-sukhesh